src/components/DarkModeToggle/DarkModeToggle.test.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+import { render } from '@testing-library/react';
+
+import DarkModeToggle from './DarkModeToggle.component';
+
+describe('DarkMode Testing', () => {
+  it('selects an element using the text content', () => {
+    const { getByText } = render(<DarkModeToggle />);
+
+    expect(getByText('Dark Mode').tagName).toBe('DIV');
```

Comment:
This works, but sometimes you may want to be less specific, in that cases you can use something like:
`expect(getByText('Dark Mode')).toBeInTheDocument()`